telematic_system/telematic_apps/web_app/server/nats_client/file_processing_nats_publisher.js

```diff
@@ -0,0 +1,15 @@
+require("dotenv").config();
+const fileProcessingSubject = process.env.FILE_PROCESSING_SUBJECT;
+
+exports.pubFileProcessingReq = async (nc, payload) => {
+  if (nc) {
+    nc.publish(fileProcessingSubject, String(payload));
```

Comment:
We should also be waiting for a response and throwing a timeout error this request wasn't acknowledged. Could potentially happen if the processing service isn't active